eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -49,40 +49,80 @@ param(
   [string]$PRBody = $PRTitle,
 
   [Parameter(Mandatory = $false)]
-  [string]$PRLabels
+  [string]$PRLabels,
+
+  [Parameter(Mandatory = $false)]
+  [string]$UserReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TeamReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$Assignees
 )
 
+. "${PSScriptRoot}\logging.ps1"
+
 $headers = @{
   Authorization = "bearer $AuthToken"
 }
+$baseURI = "https://api.github.com/repos"
+function SplitMembers ($membersString)
+{
+  return @($membersString.Split(",") | % { $_.Trim() } | ? { return $_ })
+}
 
-$query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
+function InvokeGitHubAPI($apiURI, $method, $body) {
+  $resp = Invoke-RestMethod -Method $method -Headers $headers -Body ($body | ConvertTo-Json) -Uri $apiURI -MaximumRetryCount 3
+  LogDebug "These members have been added to: https://github.com/$RepoOwner/$RepoName/pull/$prNumber"
+  ($body | Format-List | Write-Output)
+  $resp | Write-Verbose
+}
 
-function AddLabels([int] $prNumber, [string] $prLabelString)
-{
-  # Adding labels to the pr.
-  if (-not $prLabelString) {
-    Write-Verbose "There are no labels added to the PR."
-    return
+function AddReviewers ($prNumber, $users, $teams) {
+  if (!$users -and !$teams)
+  {
+    LogWarning "No users or teams provided for addition, exiting."
```

Comment:
I don't think this should be a warning, as we don't always pass in reviewers to this script. We should simply make it a no-op.